pleskdistup/common/src/leapp_configs.py

```diff
@@ -138,10 +141,13 @@ def _do_common_replacement(line: typing.Optional[str]) -> typing.Optional[str]:
     ])
 
 
-def is_repo_ok(id: typing.Optional[str], name: typing.Optional[str],
-               url: typing.Optional[str], metalink: typing.Optional[str],
-               mirrorlist: typing.Optional[str]
-               ) -> bool:
+def is_repo_ok(
```

Comment:
Too much indent, also broken alignment. Closing parenthesis should be at the same indent level as the opening. For example:
```
def func(
    param1,
    param2,
) -> R:
```
The same below.